### PR TITLE
Dominian loadout items

### PR DESCRIPTION
**Pull request by Catrly**
Adds some dominian items to the loadout. These include an NCO patch and two capes (a medical one and a military issued one). Requested by Schwann.

### CODE REVIEW

**Comment by Catrly** [comment]
!review

Not sure if I did this correctly, having loads of issues with having multiple PRs rn.

File: code/datums/uplink/gear loadout.dm [hunk 1]
```diff
@@ -214,3 +214,16 @@
...
+	desc =	"A crate containing gear for a group."
+	path =	/obj/structure/closet/crate/secure/gear_loadout/gadpathur
+	telecrystal_cost = 35
+
+
+/datum/uplink_item/item/gear_loadout/gadpathur_single
+	name =	"Gadpathur Assets (Single)"
+	desc =	"A crate containing gear for a single individual."
+	path =	/obj/structure/closet/crate/secure/gear_loadout/gadpathur/single
+	telecrystal_cost =	10
```
Comment (FluffyGhoster) [inline code/datums/uplink/gear loadout.dm:229]:
```suggestion
	telecrystal_cost = 10
```

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 2]
```diff
@@ -1052,37 +1052,108 @@
...
+	new	/obj/item/gun/projectile/automatic/rifle/dnac(src)
+	new	/obj/item/gun/projectile/automatic/rifle/dnac(src)
+	new	/obj/item/ammo_magazine/a65(src)
+	new	/obj/item/ammo_magazine/a65(src)
+	new	/obj/item/ammo_magazine/a65(src)
+	new	/obj/item/ammo_magazine/a65(src)
+	new	/obj/item/shovel/gadpathur(src)
+	new	/obj/item/shovel/gadpathur(src)
+	new	/obj/item/shovel/gadpathur(src)
+	new	/obj/item/shovel/gadpathur(src)
```
Comment (FluffyGhoster) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1142]: Single space only

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 3]
```diff
@@ -1052,37 +1052,108 @@
...
-	new /obj/item/clothing/under/gearharness(src)
-	new /obj/item/clothing/suit/armor/exclusionist(src)
-	new /obj/item/clothing/mask/exclusionist(src)
-	new /obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
-	new /obj/item/melee/chainsword(src)
+	new	/obj/item/clothing/under/gearharness(src)
+	new	/obj/item/clothing/suit/armor/exclusionist(src)
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/melee/chainsword(src)
```
Comment (FluffyGhoster) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1085]: Single space only

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 4]
```diff
@@ -1052,37 +1052,108 @@
...
+	new	/obj/item/clothing/mask/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/clothing/ears/antenna/trinary_halo/exclusionist(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
+	new	/obj/item/melee/chainsword(src)
```
Comment (FluffyGhoster) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1078]: Single space only

File: code/modules/client/preference_setup/loadout/items/suit.dm [hunk 5]
```diff
@@ -291,6 +291,8 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["dominian cape, military"]	=	/obj/item/clothing/accessory/poncho/dominia_cape/mantle
+	dominiacape["dominian cape, medical"]	=	/obj/item/clothing/accessory/poncho/dominia_cape/hospital
```
Comment (FluffyGhoster) [inline code/modules/client/preference_setup/loadout/items/suit.dm:295]:
```suggestion
	dominiacape["dominian cape, military"]	= /obj/item/clothing/accessory/poncho/dominia_cape/mantle
	dominiacape["dominian cape, medical"] = /obj/item/clothing/accessory/poncho/dominia_cape/hospital
```

File: html/changelogs/example copy.yml [hunk 6]
```diff
@@ -0,0 +1,58 @@
+################################
```
Comment (FluffyGhoster) [inline html/changelogs/example copy.yml:1]: Wrong file? Or at least it should be only one

Comment (Catrly) [reply html/changelogs/example copy.yml:1]: I have no idea why all this stuff is being added in - the commits from the other PR I did https://github.com/Aurorastation/Aurora.3/pull/19459 here

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm (hunk 2 above)
Comment (Catrly) [reply code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1142]: What does single spaces only mean?

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm (hunk 4 above)
Comment (Catrly) [reply code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1078]: No idea why this is all changed, this isn't code I've touched. Or uhhh I didn't think it was

**Pull request closed without merging**